charts/backbeat/values.yaml

```diff
@@ -1,8 +1,6 @@
 # Default values for backbeat.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-orbit:
```

Comment:
Instead of completely removing `.Values.orbit` consider moving it to `.Values.global.orbit` in each of the sub charts (the Zenko parent values will still overwrite these)

Reply:
Done! 👍